pkg/reports/store.go

```diff
@@ -24,10 +28,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+var (
+	log = ctrl.Log.WithName("store")
+)
+
 type StoreInterface interface {
```

Comment:
I don't think it matters now, but I have a feeling that as we expand this to cover more types of report this might want to go back to being a more generic Write / Read interface with less specificity to vulnerability reports.

Reply:
Yup. We'll probably refactor that one when we add Polaris or other scanner.